src/main/java/TulipTask.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class TulipTask {
+    public static void main(String[] args) {
+        String logo = "                                         \n" +
+                "--.--     |    o     --.--          |    \n" +
+                "  |  .   .|    .,---.  |  ,---.,---.|__/ \n" +
+                "  |  |   ||    ||   |  |  ,---|`---.|  \\ \n" +
+                "  `  `---'`---'`|---'  `  `---^`---'`   `\n" +
+                "                |";
+
+        ArrayList<Task> list = new ArrayList<>();
+
+        System.out.println(logo);
+
+        System.out.println("--------------------------------------------");
+        System.out.println("Hello, I'm TulipTask");
+        System.out.println("What can I do for you today?");
+        System.out.println("--------------------------------------------");
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            String input = scanner.nextLine();
+
+//            if (Objects.equals(input.toLowerCase(), "bye")) {
+//                break;
+//            }
+//
+//            if (Objects.equals(input.toLowerCase(), "list")) {
+//                listTasks(list);
+//            }
+//
+//            if (input.toLowerCase().contains("mark")) {
+//                String[] splitStr = input.split(" ");
+//                System.out.println(splitStr[0]);
+//                System.out.println(splitStr[1]);
+//            }
+
+            String command;
+
+            if (input.toLowerCase().contains("mark")) {
+                String[] splitStr = input.split(" ");
+                command = splitStr[0];
+            } else {
+                command = input.toLowerCase();
+            }
+
+            switch(command) {
+                case "list":
+                    listTasks(list);
+                    break;
+
+                case "mark":
+                    String[] splitStr = input.split(" ");
+                    int index = Integer.parseInt(splitStr[1]) - 1;
+                    markAsDone(list.get(index));
+                    break;
+
+                case "unmark":
+                    String[] split = input.split(" ");
+                    int idx = Integer.parseInt(split[1]) - 1;
```

Comment:
Consider using the long form for the variable such as "index" instead of using the short form